apps/backend/src/modules/activity-logs/activity-log.controller.ts

```diff
@@ -149,11 +157,18 @@ class ActivityLogController extends BaseController {
 	 *                   items:
 	 *                     $ref: "#/components/schemas/ActivityLog"
 	 */
-	private async findAll(): Promise<APIHandlerResponse> {
-		const activityLogs = await this.activityLogService.findAll();
+	private async findAll(
+		options: APIHandlerOptions<{
+			query: ActivityLogQueryParameters;
+		}>,
+	): Promise<APIHandlerResponse> {
+		const { endDate, startDate } = options.query;
 
 		return {
-			payload: activityLogs,
+			payload: await this.activityLogService.findAllWithParameters({
+				endDate: endDate || new Date().toDateString(),
```

Comment:
I think we always need to pass startDate in query params. And we also need to add validation for query for this endpoint
```suggestion
				endDate,
```